e2e/utils/functions/openmrs.ts

```diff
@@ -384,7 +384,7 @@ export class OpenMRS {
     await this.page.getByLabel('Application: Writes Clinical Notes').check();
     await this.page.getByRole('button', { name: 'Save Role' }).click();
     await expect(this.page.getByText('Role saved')).toBeVisible();
-    await delay(50000);
+    await delay(160000);
```

Comment:
That's a huge bump, what does it mean?

Reply:
It takes some 1 to 2 minutes the eip client for openmrs-keycloak to sync.